spring-cloud-aws-sqs/src/main/java/io/awspring/cloud/sqs/listener/QueueAttributesResolver.java

```diff
@@ -142,6 +153,16 @@ private boolean isValidQueueUrl(String name) {
 		}
 	}
 
+	private boolean isValidQueueArn(String queueArn) {
```

Comment:
Can we return Arn or null? So it can be reused in `doResolveQueueUrlFromArn` method

Reply:
@MatejNedic Thanks for the suggestion, I have amended that.